tests/libvmi/vmi.go

```diff
@@ -145,3 +145,19 @@ func WithResourceCPU(value string) Option {
 		vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceCPU] = resource.MustParse(value)
 	}
 }
+
+// WithSerialBios if set the BIOS output will be transmitted over serial.
+func WithSerialBios(secureBoot bool) Option {
+	return func(vmi *v1.VirtualMachineInstance) {
+		if vmi.Spec.Domain.Firmware == nil {
+			vmi.Spec.Domain.Firmware = &v1.Firmware{}
+		}
+		if vmi.Spec.Domain.Firmware.Bootloader == nil {
+			vmi.Spec.Domain.Firmware.Bootloader = &v1.Bootloader{}
+		}
+		if vmi.Spec.Domain.Firmware.Bootloader.BIOS == nil {
+			vmi.Spec.Domain.Firmware.Bootloader.BIOS = &v1.BIOS{}
+		}
+		vmi.Spec.Domain.Firmware.Bootloader.BIOS.UseSerial = pointer.BoolPtr(secureBoot)
```

Comment:
Please change `pointer.BoolPtr` -> `pointer.Bool`
Also, this needs to always be true (as the function name is WithSerialBios)

Reply:
Done.